java/client/src/main/java/glide/api/models/commands/geospatial/GeoSearchResultOptions.java

```diff
@@ -0,0 +1,71 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands.geospatial;
+
+import glide.api.models.commands.SortOrder;
+import java.util.ArrayList;
+import java.util.List;
+
+public class GeoSearchResultOptions {
+    /** Redis API keyword used to perform geosearch with count. */
+    public static final String COUNT_REDIS_API = "COUNT";
+
+    /**
+     * Redis API keyword used to change search behavior to return as soon as enough matches are found.
+     */
+    public static final String ANY_REDIS_API = "ANY";
+
+    private final SortOrder sortOrder;
+
+    private final long count;
+
+    private final boolean isAny;
+
+    public GeoSearchResultOptions(long count) {
+        this.sortOrder = null;
```

Comment:
As an option, use `@Builder` instead of constructors.

Reply:
`isAny` cant be used without count so i figured its easier this way.